cards/_components/Tabs.vue

```diff
@@ -0,0 +1,109 @@
+<template>
+  <div class="p-tabs">
+    <ul class="p-tabs-headers">
+      <li
+        v-for="(header, index) in headers"
+        @click="select(index)"
+        :key="index"
+        :class="['p-tabs-header', header.variant, { active: index === active }]">
+        <span>{{ header.label }}</span>
+      </li>
+    </ul>
+    <div class="p-tabs-content">
+      <slot></slot>
+    </div>
+  </div>
+</template>
+
+<script>
+export default {
+  name: 'Tabs',
+  data: () => ({
```

Comment:
@mzgajner any special reason this isn't `data() { ... }`?

Reply:
Nope, not at all, it was copypasted from parlassets. Well spotted, fixed it.